src/dev/build/tasks/os_packages/docker_generator/resources/base/bin/kibana-docker

```diff
@@ -241,10 +241,11 @@ kibana_vars=(
     xpack.alerting.rules.run.actions.max
     xpack.alerting.rules.run.alerts.max
     xpack.alerting.rules.run.actions.connectorTypeOverrides
-    xpack.alerting.maxScheduledPerMinute
```

Comment:
essentially another backport of https://github.com/elastic/kibana/pull/213487 :-) - see the `rules` version added below